packages/autocomplete-react/package.json

```diff
@@ -21,6 +24,18 @@
     "dist/"
   ],
   "dependencies": {
-    "preact": "^10.3.1"
+    "preact": "^10.3.1",
+    "@francoischalifour/autocomplete-core": "1.0.0-alpha.10",
```

Comment:
We need to loosen the dependency since it's not a dev dependency: `^1.0.0-alpha.10`. This avoids to end up having duplicate dependencies if you use a newer version of `autocomplete-core` in your app.

Reply:
a13ef0a